tests/constants/regex.js

```diff
@@ -0,0 +1,2 @@
+export const phoneNumberRegex =
+  /^[+]{1}(?:[0-9\-\\(\\)\\/.]\s?){6,15}[0-9]{1}$/;
```

Comment:
Hey @tejaskh3, I see we are not using any regex in tests. and that too from this file. 

Do we need it?